huaweicloud/services/cci/resource_huaweicloud_cci_network.go

```diff
@@ -0,0 +1,270 @@
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"namespace": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(
+						"[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"),
+						"The name can only contains lowercase characters, hyphens (-) and dots (.), and must start "+
+							"and end with a character or digit."),
+					validation.StringLenBetween(1, 200),
+				),
+			},
+			"security_group_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"network_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"vpc_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"subnet_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"cidr": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceNetworkAnnotations(d *schema.ResourceData, config *config.Config) map[string]string {
+	result := map[string]string{
+		"network.alpha.kubernetes.io/domain_id":  config.DomainID,
+		"network.alpha.kubernetes.io/project_id": config.HwClient.ProjectID,
+	}
+	if v, ok := d.GetOk("security_group_id"); ok {
+		result["network.alpha.kubernetes.io/default-security-group"] = v.(string)
+	}
+	return result
+}
+
+func resourceCciNetworkCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+	cciClient, err := config.CciV1BetaClient(region)
+	if err != nil {
+		return fmtp.DiagErrorf("Error creating HuaweiCloud CCI Beta v1 client: %s", err)
+	}
+
+	networkId := d.Get("network_id").(string)
+	subnet, err := vpc.GetVpcSubnetById(config, region, networkId)
+	if err != nil {
+		return fmtp.DiagErrorf("The subnet does not exist: %s", err)
+	}
+
+	opt := networks.CreateOpts{
+		Kind:       "Network",
+		ApiVersion: "networking.cci.io/v1beta1",
+		Metadata: networks.CreateMetaData{
+			Name:        d.Get("name").(string),
+			Annotations: resourceNetworkAnnotations(d, config),
+		},
+		Spec: networks.Spec{
+			AvailableZone: d.Get("availability_zone").(string),
+			NetworkType:   "underlay_neutron",
+			AttachedVPC:   subnet.VPC_ID,
+			NetworkID:     networkId,
+			SubnetID:      d.Get("subnet_id").(string),
+			Cidr:          d.Get("cidr").(string),
```

Comment:
subnet_id and cidr are computed, they always be empty when creating

Reply:
These arguments has been removed.